pkg/sql/sem/builtins/aggregate_builtins.go

```diff
@@ -1340,6 +1366,9 @@ const sizeOfSTMakeLineAggregate = int64(unsafe.Sizeof(stMakeLineAgg{}))
 const sizeOfSTUnionAggregate = int64(unsafe.Sizeof(stUnionAgg{}))
 const sizeOfSTCollectAggregate = int64(unsafe.Sizeof(stCollectAgg{}))
 const sizeOfSTExtentAggregate = int64(unsafe.Sizeof(stExtentAgg{}))
+const sizeOfStatementStatistics = int64(unsafe.Sizeof(appstatspb.StatementStatistics{}))
```

Comment:
Questions more for my own understanding:

We're using these `unsafe` object sizes as initial datum size estimations? (and we only use these estimations once, to set an initial size of the object?)

What does `unsafe.Sizeof` mean here? As in, how large is this?